domain/src/main/java/org/mobilitydata/gtfsvalidator/domain/entity/RawEntity.java

```diff
@@ -18,24 +18,50 @@
 
 import java.util.Map;
 
+/**
+ * The {@code RawEntity} class represents a row of a GTFS file as raw string data.
+ */
 public class RawEntity {
 
+    /**
+     * Key: header string from original CSV file
+     * Value: a String, the raw value read from the CSV file
+     */
     private final Map<String, String> contentByHeaderMap;
     private final int entityIndex;
 
+    /**
+     * @param contentByHeaderMap the object mapping rows of a GTFS .txt file on header name
+     * @param entityIndex        the 1 based index of the row
```

Comment:
I would recommend that you add documentation where this method is called from to also indicate there that the index is 1-based. It's a little strange to have the documentation here, being that the RawEntity class itself doesn't seem to care whether it is 0 or 1 based, but as long as it's also documented where the code is called from I suppose it's ok to leave this. On this topic, in this documentation, I would recommend adding a summary description that includes something like:

>This class is used by `{@link XXXX}` with a 1 based index to identify the row location within a GTFS CSV file.

Reply:
The entity is used in various use cases. We do not have dependency on these in the entity layer, therefore does not seem possible to @link them. 
We could write something like:

> This class matches a 1 based index identifying the row location within a GTFS CSV file and its content as a map of strings.

The map being described in the next @param

Reply:
@lionel-nj Ok, I think that helps. I think the important part is that the 1-based index is documented where this class is used, because that code (not this class) is what's enforcing (and relying on) the 1-based indexing.